js-miniapp-sdk/src/types/responseTypes/rewarded/index.ts

```diff
@@ -0,0 +1,16 @@
+import { AdTypes } from '../../adTypes';
+
+/**
+ * A contract declaring the reward response, consists of amount and type of the reward
+ */
+export interface Reward {
```

Comment:
Do you intend to create a `type` or an `interface`?

Reply:
Interface was in mind.